src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -22,9 +22,12 @@ public static class MSBuildLocator
         private static readonly string[] s_msBuildAssemblies =
         {
             "Microsoft.Build",
+            "Microsoft.Build.Engine",
             "Microsoft.Build.Framework",
             "Microsoft.Build.Tasks.Core",
-            "Microsoft.Build.Utilities.Core"
+            "Microsoft.Build.Utilities.Core",
+            "System.Runtime.CompilerServices.Unsafe",
+            "System.Numerics.Vectors"            
```

Comment:
These changes are already in the master branch, please rebase.

Reply:
Forgind merged master into this branch yesterday and I pulled those changes down today before committig the latest changes.